scripts/shared.js

```diff
@@ -0,0 +1,30 @@
+import ProcessData from "/scripts/ProcessData.js";
+
+const cardContainer = document.querySelector(".card-container");
+
+async function getData() {
+  const processor = new ProcessData();
+  return await processor.fetchFolderData();
+}
```

Comment:
```suggestion
const processor = new ProcessData();
async function getData() {
  return await processor.fetchFolderData();
}
```
getData가 호출될 때 마다 ProcessData클래스의 인스턴스가 생성 될 것 같은데 인스턴스마다 다른 상태가 필요 없다면 한 번만 생성해도 될 것 같아요.